README.md

```diff
@@ -28,3 +28,48 @@ aicli
 ## Example
 
 https://github.com/samuelcolvin/aicli/assets/4039449/15cc8127-c5fc-4cff-a9d7-3560f3b96d60
+
+
+## Local Installation
```

Comment:
I'm not sure I agree with this.

In general, if you have experience with python libraries, contributing should be pretty straight forward.

For general use you should install with `pip install samuelcolvin-aicli`.